chirp/drivers/ga510.py

```diff
@@ -635,7 +634,15 @@ def get_settings(self):
         dtmf = RadioSettingGroup('dtmf', 'DTMF')
 
         radioddity_settings = {
-            'language': ['English', 'Chinese'],
+            'savemode': ['Off', 'Mode 1', 'Mode 2', 'Mode 3'],
+            'cha_disp': ['CH+Name', 'CH+Freq'],
+            'chb_disp': ['CH+Name', 'CH+Freq'],
+            'txundertdr': ['Off', 'Band A', 'Band B'],
+            'rptnoiseclr': ['Off'] + ['%i' % i for i in range(100, 1001, 100)],
+            'rptnoisedet': ['Off'] + ['%i' % i for i in range(100, 1001, 100)],
+        }
```

Comment:
Why are you duplicating all of these here? If they're common to both radios, they should go in `choice_settings` to avoid the repetition, no?